tests/test_generate_files.py

```diff
@@ -11,13 +11,6 @@
 from cookiecutter import exceptions, generate
 
 
-@pytest.mark.parametrize('invalid_dirname', ['', '{foo}', '{{foo', 'bar}}'])
-def test_ensure_dir_is_templated_raises(invalid_dirname):
-    """Verify `ensure_dir_is_templated` raises on wrong directories names input."""
-    with pytest.raises(exceptions.NonTemplatedInputDirException):
-        generate.ensure_dir_is_templated(invalid_dirname)
```

Comment:
True, I removed this function because I felt it was redundant with the way the folder was found in the first place; both were using the same logic & failing the same way

I had deleted this test but did not add a new test case to exemplify this scenario, this is now fixed 🙂
I've added an additional test case in `test_find_template::template missing folder`

The error thrown will be  `NonTemplatedInputDirException`